doc/api/intl.md

```diff
@@ -184,26 +186,27 @@ to be helpful:
 - [Test262][]: ECMAScript's official conformance test suite includes a section
   dedicated to ECMA-402.
 
-[btest402]: https://github.com/srl295/btest402
```

Comment:
I like the original order better, which does not take into account capitalization and any special characters when sorting...